_authors/osci_2023/angelamao930.markdown

```diff
@@ -0,0 +1,7 @@
+---
+name: <Angela Mao>
```

Comment:
@angelamao930 Please remove the `<>` from the `name` and `github` values.

Reply:
Done!